demo/index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <script type="text/javascript">
+  var SURF_N_PERF = {
+    marks: {},
+    highResMarks: {}
+  };
+
+  SURF_N_PERF.marks.pageStart = (new Date()).getTime();
+
+  if(window.performance) {
+    if(window.performance.now) {
+      SURF_N_PERF.highResMarks.pageStart = window.performance.now();
+    }
+    if(window.performance.mark) {
+      window.performance.mark('pageStart');
+    }
+  }
+
+  SURF_N_PERF.setPageLoad = function() {
+    SURF_N_PERF.marks.loadEventEnd = (new Date()).getTime();
+
+    if(window.performance && window.performance.now) {
+      SURF_N_PERF.highResMarks.loadEventEnd = window.performance.now();
+    }
+  };
+
+  if(window.addEventListener) {
+    window.addEventListener('load', SURF_N_PERF.setPageLoad, false);
+  } else {
+    window.attachEvent('onload', SURF_N_PERF.setPageLoad);
+  }
+  </script>
+  
+  <!-- Bootstrap -->
+  <link href="demo/bootstrap.min.css" rel="stylesheet">
```

Comment:
since bootstrap.min.css is in `demo/` now, the href should just be `bootstrap.min.css`